src/Compilers/CSharp/Portable/Symbols/RangeVariableSymbol.cs

```diff
@@ -152,20 +152,21 @@ public override void Accept(CSharpSymbolVisitor visitor)
             visitor.VisitRangeVariable(this);
         }
 
-        public override TResult Accept<TResult>(CSharpSymbolVisitor<TResult> visitor)
+        public override TResult? Accept<TResult>(CSharpSymbolVisitor<TResult> visitor)
+            where TResult : default
```

Comment:
Same question for `Accept` above, at line 145

Reply:
➡️ Same answer, this one is required because the legacy visitor fails to respect the nullability of `TResult`